build_dist.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env sh
+#
+# Runs `go build` with flags configured for binary distribution. All
+# it does differently from `go build` is burn git commit and version
+# information into the binaries, so that we can track down user
+# issues.
+#
+# If you're packaging Tailscale for a distro, please consider using
+# this script, or executing equivalent commands in your
+# distro-specific build system.
+
+set -euo pipefail
+
+describe=$(git describe --long --abbrev=9)
+commit=$(git describe --dirty --exclude "*" --always --abbrev=200)
```

Comment:
why 200? I guess you just mean "something big so don't abbreviate it"?

Reply:
"I can't remember what length a sha1 is, and also I don't want it to bork when hash agility finally shows up in git", yeah.